### PR TITLE
update repo guidelines

### PR DESCRIPTION
**Pull request by TimidRobot**
## Description
update repo guidelines to include configuration settings and better support organization default community health files
- creativecommons/.github#38

## Screenshots
<details>
<summary> 🖼️  Screenshot</summary>

Screenshot 2024-03-01 at 13-16-12 GitHub Repo Guidelines — Creative Commons Open Source

</details>

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by Shafiya-Heena — APPROVED** [review]
as per my best knowledge, everything seems fine.Thank you!!

File: content/contributing-code/github-repo-guidelines/contents.lr [hunk 1]
```diff
@@ -24,57 +57,52 @@ ready for external contributors.
...
-- `/CONTRIBUTING.md`: contributor guidelines
-  - Can be project-specific or our standard [`CONTRIBUTING.md`][contributing]
-    file from [creativecommons/.github][dot-github]
-  - [Setting guidelines for repository contributors - GitHub
-    Help][setting-guidelines]
-- `/README.md`: read me information file
+- `/README.md`: repository information file
+  - begin with an H1 heading identical to repository name followed by a
+    repository description
   - Must documents how to install and build the project locally and documents a
```
Comment (possumbilities) [inline content/contributing-code/github-repo-guidelines/contents.lr:64]: Not a part of this change, but `documents` should be `document` here if you want to bundle that fix in wit this PR.

Comment (possumbilities) [reply content/contributing-code/github-repo-guidelines/contents.lr]: I corrected it.

**Comment by possumbilities** [comment]
I made two changes:
* corrected a small typo that was already there ( `documents` --> `document` )
* added a note about sorting the technologies list int the `.cc-metadata.yml` file

**Comment by joker22008** [comment]
Эоооо

**Pull request merged**